Disable random generation of trades, as this may cause trouble (duplicate trades
for wooden pickaxes that cannot be deleted).  The feature of random trades is
also not used in our worlds.

--- a/mobf_trader/config.lua
+++ b/mobf_trader/config.lua
@@ -4,8 +4,7 @@ mobf_trader.MAX_MOBS_PER_PLAYER   = 24;
 
 mobf_trader.TRADER_PRICE = 'default:gold_ingot 12';
 
--- all traders will offer a random subset of their possible trades
-mobf_trader.ALL_TRADERS_RANDOM = true;
+mobf_trader.ALL_TRADERS_RANDOM = false;
 -- whenever a random trader gets a new offer added, he will have a random number of
 -- these items in stock; the number is choosen randombly between these two values:
 mobf_trader.RANDOM_STACK_MIN_SIZE = 1;
